index.js

```diff
@@ -30,6 +30,11 @@ module.exports = exports.default = function emailPropt({
   allowInvalidChars = false
 } = {}) {
   return new Promise((resolve, reject) => {
+    // some environments (e.g., cygwin) don't provide a tty
+    if (!process.stdin.setRawMode) {
```

Comment:
You may also want to use [`tty.isatty`](https://nodejs.org/api/tty.html#tty_tty_isatty_fd).